### ISSUE TITLE
load bundle for 1 min bars returns df filled with NaN

### ISSUE THREAD

**Issue by ethan1027** [shlomikushchi/zipline-trader#170]

**Comment by shlomikushchi** [comment]
minute bars will be added in next version

**Comment by jfantell** [comment]
Hello, out of curious when will the next version (with minute frequency data) be released? :)

**Comment by jfantell** [comment]
Any update on when minute bars can be expected?

**Comment by spitzbubchen** [comment]
Yes, any updates on the minute data functionality? This is now the only thing holding me back from using zipline-trader.

**Comment by nihaoo** [comment]
same question for minute data function?
